components/datetime/benches/datetime.rs

```diff
@@ -95,6 +95,81 @@ fn datetime_benches(c: &mut Criterion) {
         true,
     );
 
+    let ten_cases = [
+        "2001-09-08T18:46:40.000[u-ca=gregory]",
+        "2017-07-13T19:40:00.000[u-ca=gregory]",
+        "2020-09-13T05:26:40.000[u-ca=gregory]",
+        "2021-01-06T22:13:20.000[u-ca=gregory]",
+        "2021-05-02T17:00:00.000[u-ca=gregory]",
+        "2021-08-26T10:46:40.000[u-ca=gregory]",
+        "2021-11-20T03:33:20.000[u-ca=gregory]",
+        "2022-04-14T22:20:00.000[u-ca=gregory]",
+        "2022-08-08T16:06:40.000[u-ca=gregory]",
+        "2033-05-17T20:33:20.000[u-ca=gregory]",
+    ]
+    .map(|s| DateTime::try_from_str(s, Gregorian).unwrap());
+
+    #[inline]
+    fn construct_any_ymd_numeric() -> DateTimeFormatter<fieldsets::YMD> {
+        DateTimeFormatter::try_new(locale!("fr").into(), fieldsets::YMD::short()).unwrap()
+    }
+
+    #[inline]
+    fn construct_fixed_ymd_numeric() -> FixedCalendarDateTimeFormatter<Gregorian, fieldsets::YMD> {
+        FixedCalendarDateTimeFormatter::try_new(locale!("fr").into(), fieldsets::YMD::short())
+            .unwrap()
+    }
+
+    group.bench_function("any/construct_and_format/ymd_numeric/10_cases", |b| {
```

Comment:
praise: thanks for including the number in the benchmark so it's easy to know you have to divide